src/Features/Core/Portable/CodeRefactorings/ExtractMethod/AbstractExtractMethodCodeRefactoringProvider.cs

```diff
@@ -59,17 +59,16 @@ protected AbstractExtractMethodCodeRefactoringProvider()
             TextSpan textSpan,
             CancellationToken cancellationToken)
         {
-            var options = document.Project.Solution.Workspace.Options;
             var result = await ExtractMethodService.ExtractMethodAsync(
                 document,
                 textSpan,
-                options,
-                cancellationToken).ConfigureAwait(false);
+                options: null,
```

Comment:
is there an overlaod that doesn't take options?  Feels like that one should eb called.  Also, if this is an internal method, we should just remove that parameter.  If it is public we should obsolete this version (IMO).